src/main/client/src/reducers/table/actions.js

```diff
@@ -558,6 +558,11 @@ export const setButtonAmount = (buttonAmount: ?number) => ({
     buttonAmount,
 });
 
+export const setActivePage = (page: Object[]) => ({
```

Comment:
Funktion parametri page ei ole array vaan pelkkä objekti?
```suggestion
export const setActivePage = (page: Object) => ({
```